src/transmit.cpp

```diff
@@ -1236,6 +1303,21 @@ void tx_send_packets(struct tx *tx, struct rtp *rtpSession, const std::vector<in
             }
         }
 
+        // If packet pacing is enabled, then we should compare the actual
+        // performance against our expected performance. If we are ahead
+        // we should wait until we're at our expected timing.
+        if(packetPace) {
+            // Get the time now
+            std::chrono::high_resolution_clock::time_point now = std::chrono::high_resolution_clock::now();
```

Comment:
Do you actually need this variable?

Couldn't you just do:
```
while (std::chrono::high_resolution_clock::now() < target) { }
```

Reply:
Agreed, it doesn't need to be there